src/main/java/ru/qeekbrains/courses/java_2/HomeWork_1/Cat.java

```diff
@@ -0,0 +1,26 @@
+package ru.qeekbrains.courses.java_2.HomeWork_1;
+
+public class Cat implements Muving {
+    private static String name;
+    private static int runDistance;
+    private static int jumpHeing;
+    public Cat(String name, int runDistance, int jumpHeing){
```

Comment:
используйте ctrl+alt+L для автоформатирования текста (отступы, пробелы)
```suggestion
    private static int jumpHeight;
    
    public Cat(String name, int runDistance, int jumpHeing) {
```